baseapp-auth/baseapp_auth/rest_framework/change_expired_password/serializers.py

```diff
@@ -45,11 +47,7 @@ def save(self, *args, **kwargs):
                 dict(new_password=[_("New password cannot be the same as the current password.")])
             )
         try:
-            # Make sure the current password is correct
-            login_serializer = LoginSerializer(
-                data=dict(email=self.user.email, password=current_password)
-            )
-            login_serializer.is_valid(raise_exception=True)
+            raise Exception()
```

Comment:
Can you add an error message to this exception and mark this TODO as a Tech Debt?
This same TODO appears in diff files. Can you add the Tech Debt mark there as well?